src/app/admin/_components/faq/hooks/useActiveFaqs.js

```diff
@@ -0,0 +1,18 @@
+import { useGetList } from 'react-admin';
+
+export const useActiveFaqs = ({ page, perPage } = { page: 1, perPage: 50 }) => {
```

Comment:
Consider putting 1 and 50 into a const variables to reduce 'magic numbers' issue. Eg:
`const DEFAULT_PAGE = 1;
const DEFAULT_PER_PAGE = 50;`

Reply:
Thanks for comment! Honestly, this function is from PR of another dev, i will notify him about this comment.

Reply:
OK